rust/src/data/ffi/mod.rs

```diff
@@ -8,14 +8,17 @@ use std::os::raw::c_char;
 use std::slice;
 
 #[cfg(feature = "polars")]
-use arrow::ffi::{ArrowArray, ArrowSchema};
+use ::polars::export::arrow;
 #[cfg(feature = "polars")]
-use polars::export::arrow;
+use ::polars::prelude::*;
 #[cfg(feature = "polars")]
-use polars::prelude::*;
+use arrow::ffi::{ArrowArray, ArrowSchema};
```

Comment:
Reminder to self: `::polars` resolves ambiguity -- It specifies the external crate instead of the internal module.